test/GithubApi.Issue.test.js

```diff
@@ -0,0 +1,73 @@
+const agent = require('superagent');
+const { expect } = require('chai');
+
+const urlApi = 'https://api.github.com';
+const namePublicRepo = 'workshop-api-testing-js';
+
+describe('Given a github user', () => {
+  let user;
+  let repository;
+
+  before(async () => {
+    const response = await agent.get(`${urlApi}/user`)
+      .auth('token', process.env.ACCESS_TOKEN)
+      .set('User-Agent', 'agent');
+
+    user = response.body;
+    const listResponse = await agent.get(user.repos_url)
```

Comment:
I would not place this constant here, i would place it where i use it

Reply:
You are placing it there to use it more globally in the test but is misplaced. It's possible the test is disorganized.